datalad/distribution/get.py

```diff
@@ -705,6 +705,23 @@ def _get_targetpaths(ds, content, refds_path, source, jobs):
         yield r
 
 
+def _check_error_reported(res: dict, error_dict: dict):
```

Comment:
```suggestion
def _check_error_reported_before(res: dict, error_dict: dict):
```
(and change invocation below) to make it a bit clear(er) on either it was reported before or just now (and thus possibly having the opposite bool meaning)